docs/docs/local-dev.md

```diff
@@ -1,26 +1,48 @@
 ---
-layout: article
-title: Using Vagrant for local development
-category: dev
+title: Local Development
 ---
 
-> Prerequisites: git, [Vagrant](https://vagrantup.com), libvirt or VirtualBox
+## Multipass
```

Comment:
Isn't vagrant is our first class dev tool currently ? I would put mutipass below the vagrant documentation.